08-components-lab/src/01-timeconverter.jsx

```diff
@@ -1,12 +1,69 @@
-/**
- * Lab 01: 
- *
- * Implement a time converter component.
- * Component should show 3 input fields labeled hours, minutes and seconds.
- * Typing a value in either field updates the other two with corresponding values.
- */
-
-import React from 'react';
-
-export default React.createClass({
-});
+/**
+ * Lab 01:
+ *
+ * Implement a time converter component.
+ * Component should show 3 input fields labeled hours, minutes and seconds.
+ * Typing a value in either field updates the other two with corresponding values.
+ */
+
+import React from 'react';
+
+export default class TimeConverter extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            sec: 0,
+            min: 0,
+            hour: 0
+        };
+    }
+
+    calcSecond = (evt) => {

```

Comment:
👍 for using arrow functions to bind methods properly. Another solution is to bind them in constructor. The benefit of binding methods in constructor is that you will get more descriptive stack trace when debugging.